functions-and-operators/string-functions.md

```diff
@@ -147,11 +147,105 @@ Return the index position of the first argument within the second argument.
+
+1. **Number (or Numeric String):** The number to be formatted. It can be a direct numeric value, a numeric string, or a number in scientific notation.
+2. **Decimal Places:** The specified number of decimal places for the returned value.
+
+**Behavior Details:**
+
+- If the first input parameter is a string and contains only digits, the function will return a result based on that number. For example, `FORMAT('12.34', 1)` and `FORMAT(12.34, 1)` yield the same result.
+- If the first input parameter is a number represented in scientific notation (using `E/e`), the function returns the result based on that number. For example, `FORMAT('1E2', 3)` returns `100.000`.
+- If the first input parameter is a string starting with non-numeric characters, the function returns zero and a warning `(Code 1292)`. For example, `FORMAT('q12.36', 5)` returns `0.00000`, but also includes a warning `Warning (Code 1292): Truncated incorrect DOUBLE value: 'q12.36'`.
+- If the first input parameter is a string mixing numbers and non-numbers, the function returns results based on the consecutive numeric part at the beginning of the parameter, and also issues a warning `(Code 1292)`. For example, `FORMAT('12.36q56.78', 1)` yields the same numeric result as `FORMAT('12.36', 1)`, but also includes a warning `Warning (Code 1292): Truncated incorrect DOUBLE value: '12.36q56.78'`.
+- If the second input parameter is negative, the function truncates the decimal part and returns an integer.
+- If any of the input parameters is `NULL`, the function returns `NULL`.
+
+**Examples:**
+
+Formatting the number 12.36 to different decimal places:
+
+```sql
+mysql> SELECT FORMAT(12.36, 1);
++------------------+
+| FORMAT(12.36, 1) |
++------------------+
+| 12.4             |
++------------------+
+```
+
+```sql
+mysql> SELECT FORMAT(12.36, 5);
++------------------+
+| FORMAT(12.36, 5) |
++------------------+
+| 12.36000         |
++------------------+
+```
+
+```sql
+mysql> SELECT FORMAT(12.36, 2);
++------------------+
+| FORMAT(12.36, 2) |
++------------------+
+| 12.36            |
++------------------+
+```
 
 ### [`FROM_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_from-base64)
 
-Decode to a base-64 string and return result.
+The `FROM_BASE64()` function is used to decode strings represented in [Base64](https://datatracker.ietf.org/doc/html/rfc4648) and returns a string in its hexadecimal form.
+
+- This function accepts a single argument: the Base64 encoded string to be decoded.
+- If the input parameter is not a valid Base64 encoded string, the `FROM_BASE64()` function will return `NULL`.
+
+> **Note:**
+>
+> The `FROM_BASE64()` function only processes valid Base64 encoded strings.
+
+**Example:**
+
+Decode the Base64 encoded string `'SGVsbG8gVGlEQg=='` (This string is the result of encoding `'Hello TiDB'`, which can be done using the [`TO_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_to-base64) function.).
```

Comment:
I think this should link to the `TO_BASE64()` description in the same document.

```suggestion
Decode the Base64 encoded string `'SGVsbG8gVGlEQg=='` (This string is the result of encoding `'Hello TiDB'`, which can be done using the [`TO_BASE64()`](#function_to-base64).
```

Reply:
Thank you for your suggestion. I had initially planned to write following the Markdown format. However, I noticed that in the current document, other titles are formatted as links. I followed the existing document format to maintain consistency.

https://github.com/pingcap/docs/blob/b5aedca6fdb4389f9a4e10e36606803354752d3e/functions-and-operators/string-functions.md?plain=1#L304